AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj

```diff
@@ -34,7 +34,8 @@
 		00C9DC6B23FA393600876B41 /* DefaultVideoTileController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 00C9DC6A23FA393600876B41 /* DefaultVideoTileController.swift */; };
```

Comment:
Thanks for PR and unit test, could you also update `CHANGELOG.md`?
You can for now put something like
```
## Unreleased
### Fixed
* <Your fix>
```